test/support/api/events.js

```diff
@@ -3,13 +3,13 @@ import { getResource } from 'helpers/json-api';
 
 Cypress.Commands.add('routeActionActivity', (mutator = _.identity) => {
   cy
-    .fixture('collections/events').as('fxEvents');
+    .fixture('test/events').as('fxEvents');
 
   cy.route({
     url: '/api/actions/**/activity*',
     response() {
       return mutator({
-        data: getResource(_.sample(this.fxEvents, 5), 'events'),
+        data: getResource(this.fxEvents, 'events'),
```

Comment:
I think I'd prefer this stay sampled for the generic use, so that every other action test isn't overly long and then when we want to test them all we can do
```js
.routeActionActivity(fx => {
  fx.data = getResource(this.fxEvents, 'events');
  return fx;
})
```